bloom-housing/bloom
# 1427/listing management preferences

# Pull Request Template

## Issue

Addresses #1427


## Description

Adds the preferences section to listings management. Check it out in partners.

The issue has some designs for separating preferences by jurisdiction but we don't currently have that data so I wasn't able to do that (unclear if that should be handled here or not, pinged the channel and commented on the issue). So for now I'm just filtering on unique preferences. Keep in mind the two housing preferences from Coliseum in our seed data are treated a little differently and purposefully do not appear on the listing page.

## Type of change


## How Can This Be Tested/Reviewed?

Create a new listing in listings management and check out the new preferences section. Save the listing then re-edit it. Preview the preferences on the listing page.

## Checklist:

Note: react-beautiful-dnd is throwing a scroll warning on drag but it doesn't seem to actually be causing any issue, and based on the activity on [the issue](https://github.com/atlassian/react-beautiful-dnd/issues/131) about it, I'm not sure there is anything I can do to remove the warning until they put in a fix
